release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -98,6 +98,9 @@ By default, the installation program now deploys a Microsoft Azure cluster using
 
 [id="ocp-4-11-networking"]
 === Networking
+==== New option for Ingress Controllers with `hostnetwork` endpoint
```

Comment:
We don't use backticks in headings: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference